.github/workflows/cibuildwheel-before-test.sh

```diff
@@ -8,12 +8,12 @@ PACKAGE_DIR=$1
 
 if [[ $(python --version 2>&1) == *"3.6."* ]]; then
   # Python 3.6 is only supported with oldest requirements
-  pip install -U -r "${PACKAGE_DIR}/.circleci/ci-oldest-reqs.txt" --progress-bar=off
+  pip install -U -r "${PACKAGE_DIR}/.github/workflows/conda-envs/oldest-test-requirements.txt" --progress-bar=off
```

Comment:
I'm not sure, but this may still trigger Dependabot. I named the old file `reqs` instead of `requirements`.